### ISSUE TITLE
Constant network errors when getting Beeper Mini set up

### ISSUE THREAD

**Issue by Dometalican** [thatmarcel/beepserv-rewrite#9]
Rooted with Chimera/Sileo on iPhone 6 using the latest a686640 (rootfull) version.

I can get the code, no problem. When I try to sign into Beeper Mini, I get a notification from the iPhone stating:

"Retrieving validation data failed because identityservicesd did not respond. Try reinstalling the tweak or restarting your device. If that fails, report this problem and try using an older version for now".

I have reinstalled, restarted, used an older version, used WiFi, 5G, and hotspot for both devices all to no avail. If it helps, my Sileo is using Libhooker as it came pre-installed with Sileo.

EDIT: In interesting developments, turns out that, after toggling the iMessage button on the iPhone on and then off again allowed this tweak to validate the registration. 

The issue now is that, when "connecting to iMessage" on Beeper Mini, it virtually goes on an eternal load without going through. It's great that it's trying harder now to go through with the phone registration but it's just not going 100% through. Would love any form of troubleshooting that can propel this forward.

**Comment by thatmarcel** [comment]
Thanks for the detailed report and sorry for the late response.

Have you tried turning iMessage on and logging into Beeper Mini for testing?

Also, just to make sure, are you using the newest version of Beeper Mini (1.2.58) from the [website](https://www.beeper.com/download)?

**Comment by Dometalican** [comment]
My friend, you are a SAINT!

I had always been using the non-beta version. After running through the process with phone registration, nothing happened so I turned on iMessage. The iPhone then popped a message saying that I was using iMessage on an iPhone called 'Beeper Mini'. I allowed it but then hit the sign in screen. I figured I put in the login info and behold, everything is working perfectly.

Thank you so much my friend. Finally got this working after two months of troubleshooting.
